django_mongodb_engine/south_adapter.py

```diff
@@ -42,7 +42,7 @@ def add_column(self, table_name, field_name, field, keep_default=True):
                 db_prep_save = field.get_db_prep_save(default, connection=connection)
                 default = connection.ops.value_for_db(db_prep_save, field)
                 # Update all the documents that haven't got this field yet
-                collection.update({name: {'$exists': False}},
+                collection.update_many({name: {'$exists': False}},
```

Comment:
@kondaiahv will this be update_many or update_one?

Reply:
That's a DDL add column method. So, that's should be update_many to add new column/field in all docs.